test/cqlpy/test_tools.py

```diff
@@ -184,6 +192,8 @@ def scylla_sstable(table_factory, cql, ks, data_dir, s3_server=None, copy_to_s3=
         yield (table, schema_file, sstables)
     finally:
         cql.execute(f"DROP TABLE {ks}.{table}")
+        for t in types:
+            cql.execute(f"DROP TYPE {t}")
```

Comment:
nitpick: I think this "finally" only covers part of the test, so in certain kinds of failure we may forget to delete the type. But since we don't expect failure, this is not a big problem.